frontend/www/js/omegaup/components/arena/ContestListv2.vue

```diff
@@ -201,8 +206,12 @@
               </template>
             </omegaup-contest-card>
           </template>
-          <template v-if="loading && !contestListEmpty">
-            <div v-for="index in 3" :key="index" class="card contest-card mb-3">
+          <template v-if="isScrollLoading">
+            <div
+              v-for="index in 3"
+              :key="'scroll-' + index"
```

Comment:
We prefer using [string interpolation](https://www.scaler.com/topics/typescript/typescript-string-interpolation/). Also, it would be good to identify each contest type to avoid duplicate indexes:
```suggestion
              :key="`current-${index}`"
```
The same should be done for the other two categories, replacing "current" with the corresponding one